app/javascript/components/App.js

```diff
@@ -0,0 +1,22 @@
+// import React from 'react';
+
+// function App() {
+//   return <h1>Hello World, component 2!</h1>;
+// }
+
+// export default App;
```

Comment:
## [OPTIONAL]
Please remove the commented/unused code to make the codebase clean and professional. 👍